PeerioTechnologies/peerio-desktop
# feat: new Signup

#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/12075/desktop-sign-up-ui-update

#### Testing instructions
New Signup flow!

* nice splash page with illustration should show if user has not previously created an account or logged in from this device
  * this will show for existing users until they log in once. may need some finessing?
* account creation process will suggest usernames to user based on their provided first and last name
  * username suggestions still kinda janky and will sometimes suggest invalid usernames (SDK fix incoming)
* account creation process now saves two user settings that it didn't before
  * subscribe to Peerio email newsletter
  * allow usage data collection
* throughout the process there's a "mock UI" on the right hand side that will update based on user input
* also check that things look good on a variety of window sizes. I've done a sweep already but worth another set of eyes.
  * big window
  * smallest window
  * wide and short
  * tall and narrow
* functionally nothing else is new, it's just split across more pages and visually nicer

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
